src/electron/views/device-connect-view/components/window-title.tsx

```diff
@@ -2,20 +2,31 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 import { NamedFC } from '../../../../common/react/named-fc';
-import { headerText, windowTitle } from './window-title.scss';
+import { actionableIconsContainer, headerText, titleContainer, windowTitle } from './window-title.scss';
 
 export interface WindowTitleProps {
     title: string;
     children?: JSX.Element;
+    actionableIcons?: JSX.Element[];
+    className?: string;
 }
 
 export const WindowTitle = NamedFC<WindowTitleProps>('WindowTitle', (props: WindowTitleProps) => {
     return (
-        <header className={windowTitle}>
-            <div>
+        <header className={[windowTitle, props.className].filter(c => c != null).join(' ')}>
+            <div className={titleContainer}>
                 {props.children}
                 <h1 className={headerText}>{props.title}</h1>
             </div>
+            {getIconsContainer(props.actionableIcons)}
         </header>
     );
 });
+
+function getIconsContainer(icons?: JSX.Element[]): JSX.Element {
```

Comment:
why outside the `NamedFC`?

Reply:
helper function.

Reply:
it's looks very specific to the component itself... plus, it's not being used anywhere else... but up to you